.github/workflows/build-subsystems.yaml

```diff
@@ -0,0 +1,114 @@
+name: Build Subsystems
+
+on:
+  workflow_call:
+    inputs:
+      dry-run:
+        description: "An indication of whether to commit/publish results"
+        required: true
+        type: string
+      is-dependabot:
+        description: "An indication of a dependabot pull request"
+        required: true
+        type: string
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      auth: ${{ steps.auth.outputs.matches }}
+      authorization: ${{ steps.authorization.outputs.matches }}
+      certificates: ${{ steps.certificates.outputs.matches }}
+      measurements: ${{ steps.measurements.outputs.matches }}
+      oidc-mock: ${{ steps.oidc-mock.outputs.matches }}
+      transfer: ${{ steps.transfer.outputs.matches }}
+    steps:
+      - name: detect auth
+        id: auth
+        uses: CodeReaper/find-diff-action@v3
+        with:
+          paths: domains/auth/
+
+      - name: detect authorization
+        id: authorization
+        uses: CodeReaper/find-diff-action@v3
```

Comment:
Should use an action in acorn repo instead

Reply:
Yes. or fork the action to our own repository. both are viable

Reply:
The action now exists locally. Documentation for the action has been added as well.

Link to commit: https://github.com/Energinet-DataHub/energy-origin/pull/1145/commits/5cb6dd4d44ff6345f6ec499baaf447fbba09c73e